src/seedu/addressbook/logic/Logic.java

```diff
@@ -85,7 +85,11 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
```

Comment:
Please update the comments for execute(). Its behavior has changed due to the overriding of isMutating() .